spec/support/factory_girl.rb

```diff
@@ -7,4 +7,14 @@
     password_confirmation 'password'
   end
 
+  factory :game do
+    user
+    sequence(:name) { |n| "Let's go Ball! {n}" }
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.